core/main.go

```diff
@@ -202,13 +214,43 @@ func main() {
 		}
 	}()
 
+	logrus.WithField("module", "main").Info("starting lcd server")
+	cfgLcd := rpcserver.DefaultConfig()
+	lcdServer, err := rpcserver.Listen("tcp://[::]:1317", cfgLcd)
+	if err != nil {
+		logrus.WithField("module", "main").Fatalln(err)
+	}
+
+	cliCtx := context.NewCLIContext().WithCodec(codec.Codec)
```

Comment:
I think @antho1404 here might be a pbm. Missing setting the client.

```
  cliCtx := context.NewCLIContext().WithCodec(codec.Codec).WithClient(rpcclient.NewLocal(node))
```
gives

```
➜  krhubert curl localhost:1317/node_info     
{"node_info":{"protocol_version":{"p2p":"7","block":"10","app":"0"},"id":"bfb2cad6e924b2154e03ca32f055ceac21ece982","listen_addr":"tcp://0.0.0.0:26656","network":"mesg-dev-chain","version":"0.32.9","channels":"4020212223303800","moniker":"c9a8ef166b9c","other":{"tx_index":"on","rpc_address":"tcp://0.0.0.0:26657"}},"application_version":{"name":"","server_name":"\u003cappd\u003e","client_name":"\u003cappcli\u003e","version":"","commit":"","build_tags":"","go":"go version go1.13 linux/amd64"}}
```

\o/

Reply:
thanks, I used directly the cosmos client that we had and it works great :)